ucfcs/GrowthPlus
# Julio/flashcards

This PR implements flashcards functionally. 

To test:
Go thru the roadmap tiles, each tile has flashcards at the end of the lesson. 
At each flashcard enter an answer, the flashcards should not take an empty input. 
Tab the flashcard, inspect there is an animation, after the animation, it correct, it should be light green, if wrong it should be red. 
Play around with them, make sure nothings breaks.
